org.develz.Crawl.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.develz.Crawl.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>crawl</name>
+  <summary>Dungeon Crawl Stone Soup</summary>
+  <description>
+    <p>
+      A roguelike adventure through dungeons filled with dangerous monsters in
+      a quest to find the mystifyingly fabulous Orb of Zot.
+    </p>
+  </description>
+  <categories>
+    <category>Game</category>
+  </categories>
+  <url type="homepage">http://crawl.develz.org/</url>
+  <launchable type="desktop-id">crawl.desktop</launchable>
+  <screenshots>
+    <screenshot>
+      <caption>The Dungeon</caption>
+      <image type="source">http://crawl.develz.org/wordpress/wp-content/uploads/2014/05/musu.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Sigmund</caption>
+      <image type="source">http://crawl.develz.org/wordpress/wp-content/uploads/2014/05/meph.png</image>
+    </screenshot>
+  </screenshots>
+  <update_contact>guillaumepoiriermorency@gmail.com</update_contact>
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">mild</content_attribute>
```

Comment:
I'm not sure if you can skip keys that have `none` value. @hughsie, is there any policy about it?

Reply:
Skipping them probably isn't a good idea (the idea for an explicit 'none' was that the question had been answered) and they compress down to almost nothing. When/if we upgrade the OARS spec with additional questions we need to paste the old attributes into the web tool and the `none` answers will hide the questions already answered.

Reply:
Thanks. Fixed it in the new repository.